README.md

```diff
@@ -1,2 +1,44 @@
 # FindYourPet
-Find Your Pet web application.
+Find Your Pet web application Install.
+here is the steps to configurate the project in your pc
+- first you must to install git in your pc.
+# GIT INSTALL 
+* **WINDOWS** :
+1. you will to go at this direccion [press here](https://git-scm.com/download/win") here you be able to install it in your pc.
+2. you will download it.
+3. you must open the executable and follow the steps of installation, when it has finised, you will search in your pc git bash and you will open it.
+
+* **MAC**
+1. you will visit git page in this direccion [press here](https://git-scm.com/download/mac), later you will download git with the steps that you find here.
+2. you will open the executable and will finish the installation
+
+* **LINUX**
+- if you're in a Debian-based distribution like Ubuntu
+*apt-get install git*
+- If you want to install Git on Linux through a binary installer, in general you can do it through the basic package management tool that your distribution brings. If you are in Fedora, you can use yum:
+*yum install git-core*
+- if you want more information, or you have another distribution you can vist the [git page](https://git-scm.com/download/linux)
+- later You can verify your version with the next command *git --version*
+# NPM AND NODE.JS INSTALL
+
+* **WINDOWS**
+you can visit this [page](https://tutobasico.com/instalar-nodejs-y-npm/) for all information about it 
+
+* **MAC** 
+if you want install NPM and NODE.JS in mac [press here](https://nodesource.com/blog/installing-nodejs-tutorial-mac-os-x/)
+
+* **Linux**
+to steps for install in linux see that [short video](https://www.youtube.com/watch?v=esNlno79dBw)
+
+# INSTALL THE PROJECT
+
+1. open git bash in windows, in mac and linux open the term, for download our project
+* go to the folder where you want that the project is there, and write 
+git clone https://github.com/remic-co/FindYourPet.git
+later to do it, you will have our project in that folder
+* in the term go to the root of the project (file FindYourPet) and run the next commad:
+*npm start*, later you will see in your term the url of our project here will run the project
```

Comment:
If you want to write a command you can use the notation for it with the "`npm start`"